diff --git a/biom/_filter.pyx b/biom/_filter.pyx
index e66f10f..b638d51 100644
--- a/biom/_filter.pyx
+++ b/biom/_filter.pyx
@@ -9,7 +9,7 @@
 from __future__ import division, print_function
 
 from itertools import compress
-from collections import Iterable
+from collections.abc import Iterable
 from types import FunctionType
 
 import numpy as np
diff --git a/biom/table.py b/biom/table.py
index 49da56b..228a1e7 100644
--- a/biom/table.py
+++ b/biom/table.py
@@ -182,7 +182,8 @@ from functools import reduce, partial
 from operator import itemgetter, or_
 from future.builtins import zip
 from future.utils import viewitems
-from collections import defaultdict, Hashable, Iterable
+from collections import defaultdict
+from collections.abc import Hashable, Iterable
 from numpy import ndarray, asarray, zeros, newaxis
 from scipy.sparse import (coo_matrix, csc_matrix, csr_matrix, isspmatrix,
                           vstack, hstack)
